pom.xml

```diff
@@ -168,6 +164,13 @@
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
+            <dependency>
+                <groupId>org.junit</groupId>
+                <artifactId>junit-bom</artifactId>
+                <version>${junit.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
```

Comment:
The JUnit BOM is already provided by the Uber parent. Shouldn't be needed here.

Reply:
Fixed